index.html

```diff
@@ -466,9 +474,33 @@ <h2>Sunset In Aurangabad</h2>
 
                 </article>
             </section>
+
+            <!-- -------------image-component----------- -->
             <section id="image" class="section">
                 <header class="section-header">Image</header>
-                <article class="container-info"></article>
+                <article class="container-info">
+                    <div class="info">
+                        <h4>Image is important to be responsive with which can be accessed easily</h4>
+                    </div>
+                    <div class="image-container">
+                        <div class="text-badge">Responsive Image</div>
+                        <div class="image-feature">
+                            <img src="image.jpg" alt="sunset" class="img-feat">
+                        </div>
+                        <div class="code">
```

Comment:
You can use better class name than code...For Eg: code-container

Reply:
Converted class name code to code-container

Reply:
changed the class name, thanks